.github/workflows/build.yml

```diff
@@ -9,6 +9,7 @@ permissions:
 jobs:
   build:
     name: Build binaries
+    if: ${{ github.repository == 'FiloSottile/age' }}
```

Comment:
This is closer to what I had in mind, but I would suggest moving this condition farther down so that binaries are always built and only the *signing* step is conditional on the source repo.